packages/create-invoice-form/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@requestnetwork/create-invoice-form",
-  "version": "0.3.0",
+  "version": "0.3.1",
```

Given the code changes, please make this a minor version increase instead of a patch.

```suggestion
  "version": "0.4.0",
```